src/eureka/S3_data_reduction/nircam.py

```diff
@@ -2,11 +2,16 @@
 import numpy as np
 from astropy.io import fits
 import astraeus.xarrayIO as xrio
-from . import sigrej, background
+from . import sigrej, background, optspex, straighten, plots_s3
 from ..lib.util import read_time, supersample
 from tqdm import tqdm
 from ..lib import meanerr as me
 
+__all__ = ['read', 'straighten_trace', 'flag_ff', 'flag_bg',
+           'fit_bg', 'cut_aperture', 'standard_spectrum', 'clean_median_flux',
+           'flag_bg_phot', 'do_oneoverf_corr', 'calibrated_spectra',
+           'residualBackground', 'lc_nodriftcorr']
+
 
 def read(filename, data, meta, log):
     '''Reads single FITS file from JWST's NIRCam instrument.
```

Comment:
I've added tasks to the project list for enabling NIRCam and NIRSpec (NRS1 + NRS2) using orders.  However, in the AI optimization project, we're seeing that different modes converge towards different optimal settings, so it may actually be detrimental to combine reductions using a single set of parameters.